packages/fx-core/templates/plugins/resource/cicd/github/README.md

```diff
@@ -0,0 +1,45 @@
+# How to use pre-cooked workflows on GitHub
+
+## Prerequisites
+- Teams app projects are version controlled by GitHub.
+- An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
+- An Azure service princial with necessary permissions.
```

Comment:
Here is the same, let customer know if they want a provision/publish pipeline, they will need Azure Service principle.